docs/release-notes/8.0.29-21.md

```diff
@@ -0,0 +1,47 @@
+# Percona XtraDB Cluster 8.0.29-21 (2022-09-)
+
+Percona XtraDB Cluster (PXC) supports critical business applications in your public, private, or hybrid cloud environment. Our free, open source, enterprise-grade solution includes the high availability and security features your business requires to meet your customer expectations and business goals.
+
+## Release Highlights
+
+Percona XtraDB Cluster is based on Percona Server for MySQL. Find a complete list of improvements and bug fixes in the [Percona Server for MySQL 8.0.29-21 (2022-08-08)](https://docs.percona.com/percona-server/latest/release-notes/8.0.29-21.html).
+
+The improvements and bug fixes for MySQL 8.0.29, provided by Oracle, and included in Percona Server for MySQL are the following:
+
+The Performance Schema tracks if a query was processed on the PRIMARY engine, InnoDB, or a SECONDARY engine, HeatWave. An EXECUTION_ENGINE column, which indicates the engine used, was added to the Performance Schema statement event tables and the sys.processlist and the sys.x$processlist views.
+
+Added support for the IF NOT EXISTS option for the CREATE FUNCTION, CREATE PROCEDURE, and CREATE TRIGGER statements.
+
+Added support for ALTER TABLE … DROP COLUMN ALGORITHM=INSTANT.
+
+An anonymous user with the PROCESS privilege was unable to select processlist table rows.
+
+Find the full list of bug fixes and changes in the [MySQL 8.0.29 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-29.html).
+
+!!! note
+
+    Percona Server for MySQL has changed the default for the [supported DDL column operations](https://dev.mysql.com/doc/refman/8.0/en/innodb-online-ddl-operations.html) to ALGORITHM=INPLACE. This change fixes the corruption issue with the INSTANT ADD/DROP COLUMNS (find more details in [PS-8292](https://jira.percona.com/browse/PS-8292).
+
+    In MySQL 8.0.29, the default setting for supported DDL operations is ALGORITHM=INSTANT. You can explicitly specify ALGORITHM=INSTANT in DDL column operations.
+
+## Bugs Fixed
+
+* [PXC-3982](https://jira.percona.com/browse/PXC-3982): A replica node is also an asynchronous slave. A SQL thread failed at started if this node was not ready to accept connections while joining the cluster. 
```

Comment:
Maybe this way? : When the replica node is also an asynchronous slave, a SQL thread failed at the start if this node ...

Reply:
Rewriteen.